src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -671,13 +678,41 @@ export class HaConfigDevicePage extends LitElement {
                   dismissText: this.hass.localize("ui.common.no"),
                 }))
               ) {
-                disableConfigEntry(this.hass, cnfg_entry);
+                let result: DisableConfigEntryResult;
```

Comment:
I think that you can actually leave the type out here. It's smart enough.

Reply:
That would make it `any`, and causes us to loose typing further down...